great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -293,17 +298,15 @@ def get_batch_data_and_markers(
                 )
 
         elif isinstance(batch_spec, (PathBatchSpec, GlueDataCatalogBatchSpec)):
-            reader_method: str = batch_spec.reader_method
-            reader_options: dict = batch_spec.reader_options or {}
-            path: str = batch_spec.path
-            schema: Optional[
-                Union[pyspark.sql.types.StructType, dict, str]
-            ] = reader_options.get("schema")
+            reader_method = batch_spec.reader_method
+            reader_options = batch_spec.reader_options or {}
+            path = batch_spec.path
+            schema = reader_options.get("schema")
 
             # schema can be a dict if it has been through serialization step,
             # either as part of the datasource configuration, or checkpoint config
             if isinstance(schema, dict):
-                schema: pyspark.sql.types.StructType = sparktypes.StructType.fromJson(
+                schema = pyspark.sql.types.StructType = sparktypes.StructType.fromJson(
```

Comment:
```suggestion
                schema = sparktypes.StructType.fromJson(
```

Reply:
@Kilo59 Thank you for spotting this pesky typo!  Fixed -- committing shortly.